docker-compose.platform.yml

```diff
@@ -114,12 +115,12 @@ services:
       - dapi_tx_filter_stream
       - dapi_envoy
     ports:
-      - 3000:80 # JSON RPC and gRPC Web
-      - 3010:50051 # gRPC Native
+      - ${PLATFORM_DAPI_NGINX_HTTP_PORT:?err}:80 # JSON RPC and gRPC Web
+      - ${PLATFORM_DAPI_NGINX_GRPC_PORT:?err}:50051 # gRPC Native
     volumes:
-      - ./configs/${NETWORK:?err}/dapi/nginx/default.conf:/etc/nginx/conf.d/default.conf
-      - ./configs/${NETWORK:?err}/dapi/nginx/grpc.conf:/etc/nginx/conf.d/grpc.conf
-      - ./configs/${NETWORK:?err}/dapi/nginx/errors.grpc_conf:/etc/nginx/conf.d/errors.grpc_conf
+      - ~/.mn/${CONFIG_NAME:?err}/default.conf:/etc/nginx/conf.d/default.conf
```

Comment:
Let's export `MN_HOME_DIR` too?

Reply:
How? I spent 3 hours fighting awilix to try and access the container to get this var, but it just doesn't work. Can you teach me how to use a debugger so I can see what the actual path is for vars like this? At the moment I am just guessing :(

Reply:
To use debugger you should add `bin/mn` as you run script in run/debug settings and specify a needed command as arguments. I've attached a screenshot how it looks in WebStorm.
<img width="1008" alt="Screenshot 2020-11-03 at 20 13 33" src="https://user-images.githubusercontent.com/24296/98030359-f3532a80-1e21-11eb-912e-723693fffaee.png">
I guess in VC must be about the same.

Reply:
Yes, it's exactly the same. I have no problems starting the debugger and using it. I just can't find the awilix container data, so I don't understand how to access it in different contexts.

Reply:
Ah, I see. So it depends on how you register service in the container. If you use `asClass` then dependencies will be passed to `constructor` params which are the same as service names in the container. In case of services registered using `asFunction` then dependencies will be passed as function params. We have a special case with `runWithDependecies` in command classes. In `BaseCommand` we manually pass args, options, and dependencies to `runWithDependecies` https://github.com/dashevo/mn-bootstrap/blob/47e12a8b85fa51db3b1d1e326e8e9dab7cb5c5c4/src/oclif/command/BaseCommand.js#L99-L103